### ISSUE TITLE
Missing projects in the API

### ISSUE THREAD

**Issue by Ly0n** [ecosyste-ms/ost#77]
I did a random check to see if any projects were missing, as the total number of projects seemed a bit low to me. Here are the projects I could find so far whose issues we should be getting, but which are not visible in the API:

- [x] https://github.com/SciTools/iris/issues?q=is%3Aopen+label%3A%22Good+First+Issue%22+sort%3Aupdated-desc
  - fixed bug in case sensitivity issue in label name
- [ ] https://github.com/Unidata/MetPy/issues?q=is%3Aopen+label%3A%22good+first+issue%22+sort%3Aupdated-desc
  - issues created within the past two years are showing up now 
- [x] https://github.com/pybamm-team/PyBaMM/issues?q=is%3Aopen+label%3Ahacktoberfest+sort%3Aupdated-desc
- [ ] https://github.com/hubblo-org/scaphandre/issues?q=is%3Aopen+label%3A%22help+wanted%22+sort%3Aupdated-desc
  - most of these issues where created over 2 years ago
- [x] https://github.com/electricitymaps/electricitymaps-contrib
- [ ] https://github.com/PyPSA/PyPSA/issues?q=is%3Aopen+label%3A%22help+wanted%22+sort%3Aupdated-desc
  - these issues are all over 2 years old
- [ ] https://github.com/nsidc/earthaccess/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22
  - this project doesn't appear to be listed in the opensustain.tech readme?
- [x] https://github.com/meteostat/meteostat-python/issues?q=is%3Aopen+label%3A%22help+wanted%22+sort%3Aupdated-desc
- [x] https://github.com/FlexMeasures/flexmeasures/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
  - fixed a bug with syncing older issues into the system
- [ ] https://github.com/ev-map/EVMap/issues/133
  - this issue is over 2 years old

**Comment by Ly0n** [comment]
> [Unidata/MetPy/issues (is:open label:"good first issue" sort:updated-desc)](https://github.com/Unidata/MetPy/issues?q=is%3Aopen+label%3A%22good+first+issue%22+sort%3Aupdated-desc)
issues created within the past two years are showing up now

Is it possible that you are using the time of issue creation rather than than the last_updated to measure activity?

**Comment by andrew** [comment]
@Ly0n yes, that's what I've been doing the whole time, I can change the filter to use `updated_at` instead.

**Comment by andrew** [comment]
I've deployed an update using `updated_at` instead of created_at, but quite a few very old things show up like this: https://github.com/streetmix/streetmix/issues/298

**Comment by Ly0n** [comment]
- [ ]  https://github.com/catalyst-cooperative/pudl/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen+label%3Agood-first-issue  -> Using a dash inbetween.

**Comment by andrew** [comment]
> * [ ]   https://github.com/catalyst-cooperative/pudl/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen+label%3Agood-first-issue  -> Using a dash inbetween.

just deployed a fix for that one
